app/internal/friend_view.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from sqlalchemy.orm import Session
+
+from app.routers.event import sort_by_date
+from app.routers.user import get_all_user_events
+
+
+def get_events_per_friend(
+        session: Session,
+        user_id: int,
+        my_friend: str,
+) -> List:
```

Comment:
wrong annotation, should be List[str]

Reply:
I changed the annotation. But to List[Event] because it returns a list of events.